cpp/density/SphereVoxelization.cc

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2010-2020 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#include <cmath>
+#include <stdexcept>
+
+#include "SphereVoxelization.h"
+
+/*! \file SphereVoxelization.cc
+    \brief Routines for computing voxelized densities from spheres centered at points.
+*/
+
+namespace freud { namespace density {
+
+SphereVoxelization::SphereVoxelization(vec3<unsigned int> width, float r_max)
+    : m_box(box::Box()), m_width(width), m_r_max(r_max)
+{
+    if (r_max <= 0.0f)
+        throw std::invalid_argument("SphereVoxelization requires r_max to be positive.");
+}
+
+//! Get a reference to the last computed voxels
+const util::ManagedArray<unsigned int>& SphereVoxelization::getVoxels() const
+{
+    return m_voxels_array;
+}
+
+//! Get width.
+vec3<unsigned int> SphereVoxelization::getWidth()
```

Comment:
```suggestion
const vec3<unsigned int> SphereVoxelization::getWidth() const
```

Reply:
What is the need for the first `const` here? This returns a copy of `m_width`, meaning the user can't modify the data in the in the `SphereVoxelization` class, so why do we care whether or not the user can change the returned copy?

Reply:
You are correct, this is a mistake on my part. Ignore this, as well as the corresponding suggestions on the pxd file.

Reply:
You are correct. For future reference, I had accidentally added a const qualifier to the return type, which is meaningless here. In the interest of being able to batch commit these suggestions, I've updated the suggestion here, in the header, and in the corresponding pxd file.